bin/ci/ubuntu/build-and-test.sh

```diff
@@ -66,11 +66,13 @@ fi
 # if we get a crash.
 gdb -return-child-result -quiet -batch \
     -ex "set env MALLOC_CHECK_=3" \
+    -ex "set demangle-style none" \
     -ex "set print thread-events off" \
+    -ex "file $APP_PATH" \
+    -ex "set args $APP_ARGS" \
     -ex run \
     -ex "thread apply all backtrace full" \
     -ex "quit" \
-    --args $APP_PATH $APP_ARGS
```

Comment:
On Travis, the following two symbols in this build cause `c++filt` and `gdb` to crash. Setting the demangle style to none prevents the crash.
```_ZSt7forwardIRKZN6ripple10BaseWSPeerINS0_16ServerHandlerImpENS0_9SSLWSPeerIS2_EEE3runEvEUlRT_E_EOS6_RNSt16remove_referenceIS6_E4typeE
_ZSt7forwardIRKZN6ripple10BaseWSPeerINS0_16ServerHandlerImpENS0_9SSLWSPeerIS2_EEE3runEvEUlRT_E_EOS6_RNSt16remove_referenceIS6_E4typeE```